App/features/photos/reducer.ts

```diff
@@ -21,7 +23,14 @@ export interface PhotosState {
 
 export type PhotosAction = ActionType<typeof actions>
 
-export default combineReducers<PhotosState, PhotosAction>({
+const persistConfig: PersistConfig = {
+  key: 'photos',
+  storage: AsyncStorage,
+  whitelist: ['queryData', 'processingPhotos'],
```

Comment:
Another nested persist config that persists the camera roll last queried time and processing photos, but not the camera roll photos queried from the Textile node.